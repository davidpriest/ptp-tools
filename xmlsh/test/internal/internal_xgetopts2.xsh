# test of xgetopts

set -- -longa "Option A" -bc "Option BC" -bool -multi "Multi 1" -multi <[ <multi>Multi 2 as XML</multi> ]> Remaining <[ <args>Args</args> ]>

# Test with spaces before and after the , and :
PASS=$(xgetopts -a -o "a=longa:, multi:+ , missing: " -p "bool, bc: " -- "$@")
shift $?
echo PASS $PASS
echo "$@"

