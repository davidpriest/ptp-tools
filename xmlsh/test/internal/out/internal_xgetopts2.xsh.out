PASS -bool -bc Option BC
Remaining <args>Args</args>
